mods/ra/maps/allies-09a/allies09a.lua

```diff
@@ -0,0 +1,119 @@
+   information, see COPYING.
+]]
+
+lstReinforcements =
+{
+	actors = { "mcv"},
+	entryPath = { AlliedMCVEntry.Location, Unload1.Location },
+	exitPath = { AlliedMCVEntry.Location }
+}
+
+ExtractionHelicopterType = "tran"
+ExtractionPath = { HeliWP01.Location, HeliWP02.Location, HeliWP03.Location }
+
+kosyginType = "gnrl"
+contectKosygin = false
+
+MissionAccomplished = function()
+	Media.PlaySpeechNotification(Greece, "MissionAccomplished")
+end
+
+MissionFailed = function()
+	Media.PlaySpeechNotification(Greece, "MissionFailed")
+end
+
+InitialAlliedReinforcements = function()
+	Trigger.AfterDelay(DateTime.Seconds(1), function()
+		Media.PlaySpeechNotification(Greece, "ReinforcementsArrived")
+		Reinforcements.ReinforceWithTransport(Greece, "lst.reinforcement", lstReinforcements.actors, lstReinforcements.entryPath, lstReinforcements.exitPath)
+	end)
+end
+
+RescueFailed = function()
+	Media.PlaySpeechNotification(Greece, "ObjectiveNotMet")
+	Greece.MarkFailedObjective(KosyginSurviveObjective)
+end
+
+CreateKosygin = function()
+	Greece.MarkCompletedObjective(UseSpyObjective)
+	Media.PlaySpeechNotification(Greece, "ObjectiveMet")
+	kosygin = Actor.Create(kosyginType, true, { Location = KosyginSpawnPoint.Location, Owner = Greece })
+	Trigger.OnKilled(kosygin, RescueFailed)
+	ExtractObjective = Greece.AddPrimaryObjective("Extract Kosygin.")
+	Trigger.AfterDelay(DateTime.Seconds(1), function() Media.PlaySpeechNotification(Greece, "TargetFreed") end)
+end
+
+InfiltrateForwardCenter = function()
+	Trigger.OnInfiltrated(USSRFC, function()
+		if contectKosygin == false then
+			contectKosygin = true
+			CreateKosygin()
+		end
+	end)
+
+	Trigger.OnKilledOrCaptured(USSRFC, function()
+		if not Greece.IsObjectiveCompleted(UseSpyObjective) then
+			Greece.MarkFailedObjective(UseSpyObjective)
+		end
+	end)
+end
+
+Tick = function()
+	USSR.Cash = 5000
+	if Greece.HasNoRequiredUnits() then
+		USSR.MarkCompletedObjective(USSRObj)
+	end
+end
+
+TriggerRevealUSSRBase = function()
+	Trigger.OnEnteredProximityTrigger(LowerBaseWP.CenterPosition, WDist.FromCells(10), function(a, id)
+		if a.Owner == Greece then
+			Trigger.RemoveProximityTrigger(id)
+			local cam = Actor.Create("Camera", true, { Owner = Greece, Location = revealLowerBase.Location })
+			Trigger.AfterDelay(DateTime.Seconds(15), cam.Destroy)
+		end
+	end)
+	Trigger.OnEnteredProximityTrigger(UpperBaseWP.CenterPosition, WDist.FromCells(10), function(a, id)
+		if a.Owner == Greece then
+			Trigger.RemoveProximityTrigger(id)
+			local cam = Actor.Create("Camera", true, { Owner = Greece, Location = revealUpperBase.Location })
+			Trigger.AfterDelay(DateTime.Seconds(15), cam.Destroy)
+		end
+	end)
+end	
+
+TriggerExtractKosygin = function()
+	Trigger.OnEnteredProximityTrigger(KosyginExtractPoint.CenterPosition, WDist.FromCells(10), function(actor, triggerflee)
+		if actor.Type == kosyginType then
+			heli = Reinforcements.ReinforceWithTransport(Greece, ExtractionHelicopterType, nil, ExtractionPath)
```

Comment:
`heli` is not used.

Reply:
Fixed at [c9bbea5](https://github.com/OpenRA/OpenRA/commit/c9bbea5b361b63990c5c073f6ac271692cc2ea62)